wiki/en/misc/1-index.html

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jamulus ‒ Play music online. With friends. For free."
+title: "Jamulus — Play music online with friends."
 lang: "en"
 permalink: /
 layout: mainhomepage
 ldjAppCategory: "Music"
-ldjSlogan: "Play music online. With friends. For free."
+ldjSlogan: "Play music online with friends."
 ldjCreator: "Volker Fischer and contributors"
-metadescription: "Jamulus is open source software which enables musicians to perform music, rehearse or just jam, all in real time over the Internet."
+metadescription: "Copylefted libre (<a href="https://wikiless.org/wiki/GNU_General_Public_License?lang=simple" target="_blank" rel="noreferrer">GPLv3+</a>) software for musicians to perform, rehearse or just jam together — all in real-time over the Internet."
```

Comment:
Get rid of the HTML here so you can build the site locally and can see what it all looks like. Reviewing this stuff on Github is not easy.

Strictly speaking it's GPL2+ BTW.

Reply:
No hyphen in "real time". It's a noun here, not an adjective.